public/omni/infrastructure-and-extensions/install-airgapped-omni.mdx

```diff
@@ -417,7 +417,28 @@ Guides on creating a cluster on Omni can be found here:
 
 Because we're working in an airgapped environment we will need the following values added to our cluster configs so they know where to pull images from. More information on the Talos MachineConfig.registries can be found [here](https://www.talos.dev/latest/talos-guides/discovery/).
 
-> **NOTE:** In this example, cluster discovery is also disabled. You may also configure cluster discovery on your network. More information on the Discovery Service can be found [here](https://www.talos.dev/latest/talos-guides/discovery/)
+<Note>
+By default, Talos uses the public discovery service at:
+
+```
+https://discovery.talos.dev
+```
+
+In air-gapped or restricted environments, you must take one of the following actions:
+
+- Disable cluster discovery (`cluster.discovery.enabled: false`)
```

Comment:
there is no point in doing that actually, as Talos doesn't break if discovery.talos.dev is unavailable, it might be disabled to avoid log messages, but there will be no functional change